wren-ai-service/src/utils.py

```diff
@@ -101,8 +106,38 @@ def wrapper_timer(*args, **kwargs):
                     f.write("\n-----------------------\n")
                     f.write(test_records[-1])
 
-            return value
+            return result
 
         return func(*args, **kwargs)
 
     return wrapper_timer
+
+
+def async_timer(func):
+    load_env_vars()
```

Comment:
removing invoking the loading env vars function can reduce the impact on the function with this decorator.